bin/run-tests-in-docker.sh

```diff
@@ -1,9 +1,27 @@
-#! /bin/bash -e
+#!/usr/bin/env sh
 
```

Comment:
Like `run-in-docker.sh`, this file should actually have `set -e` to ensure that a failure to build the Docker image fails the script.